includes/widgets/image-box.php

```diff
@@ -270,12 +270,22 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Border::get_type(),
+			[
+				'name' => 'image_border',
+				'selector' => '{{WRAPPER}} .elementor-image-box-wrapper img',
+				'separator' => 'before',
+			]
+		);
+
 		$this->add_responsive_control(
 			'image_border_radius',
 			[
 				'label' => esc_html__( 'Border Radius', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
 				'size_units' => [ 'px', '%' ],
+				'separator' => 'after',
```

Comment:
Why add this separator?

Reply:
Separate all the border's controls, Dor approved.